test/suite/test_cc05.py

```diff
@@ -77,9 +77,9 @@ def test_cc(self):
         # Move stable to 35 so there's something to checkpoint.
         self.conn.set_timestamp('stable_timestamp=' + self.timestamp_str(35))
 
-        # Trigger checkpoint cleanup and wait until it is done.
-        ckpt_name = "checkpoint_one" if self.named else ""
-        self.check_cc_stats(ckpt_name=ckpt_name)
+        # Trigger checkpoint.
```

Comment:
@kommiharibabu, I removed the obsolete cleanup check because we are not supposed to clean anything at this point of the test. Do you agree?

Reply:
Here also, we can have obsolete pages before of updates earlier. So we can still need checkpoint cleanup. Is it causing any problem if we keep the checkpoint cleanup.

Reply:
Are you sure? Oldest is pinned to 10 line 69 and updates start from ts 20.

Reply:
Nice, finding. Yes, we do not need the cleanup.